src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -92,12 +92,16 @@ extern "C" {
 
 using namespace deluge::gui;
 
```

Comment:
added UI_MODE_NOTES_PRESSED to the auditionPadAction and mutePadAction UI arrays so that you can use mute and audition pads while holding down a note (or in this case, an automation lane)